app.js

```diff
@@ -38,7 +41,7 @@ app.use((err, req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
-  if (err.msg === "Article not found") {
+  if (err.msg === "Article not found" || err.msg === "Comment not found") {
     res.status(404).send(err);
   } else {
     next(err);
```

Comment:
As psql errors dont have a msg key, this if statement can be simplified to just check for err.msg